dashboard/pages/artists.py

```diff
@@ -27,18 +27,21 @@ def show_artists():
         '1 day', '1 week', '1 month', "1 year"], horizontal=True)
 
     pop_artists = get_popular_artists(conn, timeframe)
-    st.header("Top Artists")
```

Comment:
Slightly hilarious, but I prefer the previous code where there were capitalisation

Reply:
Bring that up with the project manager